smart-contracts/contracts/BridgeBank/BridgeBank.sol

```diff
@@ -574,4 +717,23 @@ contract BridgeBank is BankStorage,
 
         emit LogUnlock(recipient, token, amount);
     }
+
+    function setBridgeTokenDenom(
+      address _token, string memory _denom
+    ) public onlyOperator returns (bool) {
+      contractDenom[_token] = _denom;
+      return BridgeToken(_token).setDenom(_denom);
+    }
+
+    function batchSetBridgeTokenDenom(
```

Comment:
Same comment, remove the string from the parameter to this function, and remove the permissions such that anyone can call the function.